whatsapp.py

```diff
@@ -1,185 +1,27 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+from parsing_tools import *
```

Comment:
maybe I'm wrong, but isn't the convention is to first import global libraries, and then import localised?

Reply:
Correct, I was unaware of this convention